projects/core/styles/mixins/textfield.less

```diff
@@ -288,6 +290,11 @@
 
     .t-input {
         .textfield-input();
+        .ios-only({
+            &:focus { // disable zoom
+              font-size: 16px;
```

Comment:
We cannot do that, it would break the design. Large inputs are already 16px, small ones would look bad this way.